src/duct/__main__.py

```diff
@@ -451,11 +452,24 @@ def from_argv(cls) -> Arguments:  # pragma: no cover
             "Leading directories will be created if they do not exist. "
             "You can also provide value via DUCT_OUTPUT_PREFIX env variable. ",
         )
+        parser.add_argument(
+            "--summary-format",
+            type=str,
+            default=os.getenv("DUCT_SUMMARY_FORMAT", EXECUTION_SUMMARY_FORMAT),
+            help="File string format to be used when printing the summary following execution.",
```

Comment:
```suggestion
            help="Output template to use when printing the summary following execution.",
```

What's a "file string format," anyway?